lib/db_utils.py

```diff
@@ -0,0 +1,59 @@
+import sqlite3
+from typing import Tuple
+
+import pandas as pd
+from sqlalchemy import create_engine
+
+from lib.constants import SQL_DIR, BASE_DIR
+
+
+def drop_and_initialize_tables(path_to_db):
+    """Init the tables of our DB, setting primary keys.
+    Need to do this up front with SQLite, cannot ALTER to add primary keys later.
+    """
+
+    connection = sqlite3.connect(path_to_db)
+
+    with open(SQL_DIR / "init.sql") as f:
+        query = f.read()
+
+    with connection:
+        connection.executescript(query)
+        connection.commit()
+
+
+class DbRepository:
+    def __init__(self, db_path: str):
+        self.db_path = db_path
+        self.engine = create_engine(f"sqlite:///{self.db_path}", echo=False)
+
+    def get_data_for_time_range(
+        self, start_time: str, end_time: str
+    ) -> Tuple[pd.DataFrame, pd.DataFrame]:
+        # Cannot set table name as an SQL param:https://stackoverflow.com/questions/46736633/syntax-error-with-python3-and-sqlite3-when-using-parameters
+        raw_query = lambda x: f"select * from {x} where timeFrom BETWEEN ? AND ?"
```

Comment:
im suprised you can write a f string to make this query

Reply:
but I want it to be flexible to insert `fpn` or `boal`, hence the lambda with f-string combo - is there a better way?

Reply:
ah, I see, yea then lambda is fine. What about the ?, ?. 
Can you just put the strings in there, you might need some 'single quote'

This is small, so feel free to ignore.